### PR TITLE
File structure

### PR DESCRIPTION
**Pull request by Fatmasiam**
Relates : #9

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,5 +18,9 @@
   "bugs": {
     "url": "https://github.com/FACG6/Gaza-Freelancing-Community/issues"
   },
-  "homepage": "https://github.com/FACG6/Gaza-Freelancing-Community#readme"
+  "homepage": "https://github.com/FACG6/Gaza-Freelancing-Community#readme",
+  "dependencies": {
+    "dotenv": "^7.0.0",
```
Comment (kannanhassouna) [inline package.json]: unnecessary modules

**Review by kannanhassouna — CHANGES_REQUESTED** [review]

**Review by kannanhassouna — CHANGES_REQUESTED** [review]
in ``` public/js/helper/dom.js ``` the name of the file should descriptive that file will help in what

**Comment by kannanhassouna** [comment]
thank you for editing ``` file structure ``` issue

**Comment by kannanhassouna** [comment]
You should include package-lock.json file in your P.R

**Review by kannanhassouna — CHANGES_REQUESTED** [review]
Add ``` .env ``` to .gitignore file

**Comment by Fatmasiam** [comment]
> Add `.env` to .gitignore file

done

File: package.json (hunk 1 above)
Comment (Fatmasiam) [reply package.json]: I remove it

**Comment by Fatmasiam** [comment]
> You should include package-lock.json file in your P.R

but that's just file structure !!  @knanahassouna1

**Review by kannanhassouna — APPROVED** [review]

**Review by Angham116 — APPROVED** [review]

**Review by Anies12 — APPROVED** [review]

**Review by ahmedalmi95 — APPROVED** [review]

**Pull request merged**